btsieve/tests/test_data/find_transaction_in_missing_block/transaction.hex

```diff
@@ -0,0 +1 @@
+020000000001024facc5f3eacd84a66b6e4b6a67a0083c2e7aa373a632e6c157dd9a8a19e17d4e0100000017160014b0d841f1f3f262601a40d7fd38483537cf0b2745feffffff1dad244cd6462c045c4eddf37552bb53739c24067174c4c8e82533ebd00cade4000000001716001464c6c98da700017cbb244120aa5acfd5756674e5feffffff0200cdf5050000000017a914655058043be28697efceabe405541dd346eac2a78700e1f5050000000017a914b31e69596d6838cbc30139bf9ca53a9482c51444870247304402203af9df942a7c23380eecbc4f229e6a3d8caeadc3d50e6c2ef70c5bbf6fade93a02200aa09e6c56b85020efd91cba16de9f01111d2308296ff5fb408f56f1bef84f42012102811419de9710ecc41207e1d03698c6c4b416c53602b2aea9e7279e0d583a1bdf0247304402205a9f1a5c6c7dd73e74fe12fd4e6af8c1512fd329ede8e3f06f5d1a66bfaf0e2a02200fe57021fcc606530bbba64c50f2239c122feb63653d43a4dcdaa0f2f293c4e50121028396603687ad920e9b43654a49d3c0116abc7138dd975a16de2bdd2109af3f5493010000
```

Comment:
So was this generated by "generate_test_data.sh`?

Reply:
The scripts do generate all the test data. We also check in all the test data, so you don't have to run the scripts every time you run the tests.

Reply:
Please document that somewhere.